app/javascript/mastodon/features/list_editor/components/edit_list_form.js

```diff
@@ -26,6 +28,7 @@ class ListForm extends React.PureComponent {
   static propTypes = {
     value: PropTypes.string.isRequired,
     disabled: PropTypes.bool,
+    isExclusive: PropTypes.bool,
     intl: PropTypes.object.isRequired,
     onChange: PropTypes.func.isRequired,
     onSubmit: PropTypes.func.isRequired,
```

Comment:
```suggestion
    onSubmit: PropTypes.func.isRequired,
    onToggle: PropTypes.func.isRequired,
```
Missing PropType validation

Reply:
Fixed in 5f1c86e5bb0f4a2dd4289c98560df22cffc37cf7